mailpoet/assets/css/src/components-automation-editor/_step.scss

```diff
@@ -33,3 +33,19 @@
 .mailpoet-automation-editor-step-subtitle {
   color: inherit;
 }
+
+.colored-icon {
+  box-sizing: content-box;
```

Comment:
I'm mentioning a problem with sizing below in the JS component. Not sure if the `content-box` sizing is related.